### ISSUE TITLE
Replicate measurement line

### ISSUE THREAD

**Issue by JaneGodiksen** [ices-eg/SmartDots#217]
It would be very useful a new measurement system. The reader must be able to create a line and replicate parallel lines to this first one. The measurements must be the distances between these two lines.
(comment from Pedro Lino following a swordfish calibration exercise)

**Comment by JCoadDavies** [comment]
This needs clarification but I think this is possible already as multiple lines can be drawn on an image

**Comment by JaneGodiksen** [comment]
Someone needs to contact Pedro and hear what he means and why this is important?

**Comment by JaneGodiksen** [comment]
I'll add a first milestone to February meeting. Then we can change when we have a clarification.

**Comment by JCoadDavies** [comment]
I have written to Pedro for clarification

**Comment by JaneGodiksen** [comment]
After talking to Pedro, it is clear that the vertebra needs quite a few lines. 
There are several issues:
1) The green line in the picture is the first lines placed (and it is placed by the coordinator). The red line is the reading line. This needs to be perpendicular to the first line. 
My suggestion is that the coordinator place these lines on the image BEFORE it is uploaded to SmartDots. And then the coordinator create a fixed reading line on top of the red line. Then the dots are secured on the correct line. 
Is it possible to make these kind of lines in the software? Getting exactly 90 degrees?
2) The readers prefer to place lines instead of dots (yellow lines). Is it possible to get this into the software? They will also need to be perpendicular to the reading line. 

![Vertebra ageing Pedro Lino](https://user-images.githubusercontent.com/26567257/111995691-806cf680-8b19-11eb-8ac2-3e0b43053230.jpg)

**Comment by wimallegaert** [comment]
It is for sure possible to integrate these functionalities in the Smartdots software but this is not an easy request. Changes will have an impact on the SmartDots software, Web API, ICES database and reporting. To be able to calculate the distance between the green and yellow line. The green line should also be drawn in the Smartdots software.

SmartDots software:
- integrate new types of lines
- add functionality to draw lines in 90 degrees in relation with the reading line (red)

Web API
- new interface property: types of lines

ICES database
- new interface property: types of lines

Reporting
- calculate distances between the lines

**Comment by JCoadDavies** [comment]
Q1 2022 NEED TO DECIDE ON THIS

**Comment by JCoadDavies** [comment]
Agreed at the meeting to push this into the future when we have more time and resources!

**Comment by wimallegaert** [comment]
To make it a less complex change, we could add an extra setting in the SmartDots client so users can choose when they add a point to the readling line it can be shown as a dot on the reading line or as a line perpendicular to the reading line. The setting will not be stored in the database.

Patricia will check this proposal with Pedro.

**Comment by gonpatricia** [comment]
Pedro agreed that this proposal is a good solution. 

In response to this, also make some further suggestions:
1.    Creating more types of non counting marks, for exemple having an edge mark, a pre-growth structure mark and a vascularization mark. These would the be reported in the distances file with a code for example E=edge.
2.    Also if should be possible to download the report with the "non accepted age" as we usually do intra-reader comparisons (e.g. comparing the first reading with the second reading of the same reader to check for consistency), maybe this could be an option upon the download something like - include the non-accepted reading in the report file? Y or N.
